### ISSUE TITLE
Request: Patreon - Support for other post types

### ISSUE THREAD

**Issue by SierraKomodo** [mvdicarlo/postybirb-plus#52]
TL;DR: Add support for the Link, Video, Audio, and Livestream post types under Patreon for submissions.

---

Currently, Postybirb+ supports posting Text and Image posts on Patreon, from what I can tell. Providing support for posting the other post types available through Patreon's submission panel (Video, Livestream, Audio, and Link) would be a nice feature to have. I would suggest adding this as a drop-down selection option in the edit screen for submissions in Postybirb+ and, for post types that require additional information (I.e., the URL for a link type post), include additional fields based on the selected post type.

Personally, I would primarily be making use of the 'Link' post type due to my use of Postybirb in cross-posting writing and worldbuilding updates to various social platforms that link back to the main website I post my written work onto.

**Note**: I'm not including the 'Poll' type as this would probably be better off as a separate feature, due to other platforms also having polls - Also, cloning the same poll across multiple platforms isn't the _best_ idea as it can skew results if some of your followers use multiple platforms to vote.

**Comment by jerrycurly** [comment]
I would like this feature too. If someone pointed me in the right direction, I could look into it myself. Specifically, I would like to post videos. Currently, it allows videos, but it posts them as images, and they don't display, so it would have to see the filetype and post to the appropriate submission path.
